### DISCUSSION TITLE
Add colors to Flags, Global Flags, Usage, Topics

### DISCUSSION THREAD

**Discussion by AllanOricil** [forcedotcom/cli#2501] [Ideas]
Do something like Volta. It can be a feature flag if people don't like it.

<img width="867" alt="image" src="https://github.com/forcedotcom/cli/assets/55927613/354c312e-b611-4b6f-8c5a-a74a5bbbe108">

It looks better for my eyes than just using white and grey colors

<img width="861" alt="image" src="https://github.com/forcedotcom/cli/assets/55927613/34fe0e70-046f-45e7-aa84-1aeabec00ebc">

**Comment by AllanOricil** [comment]
@mshanemc this is also oclif scope right?

**Reply by mshanemc** [reply]
yeah, that'd probably be in here: https://github.com/oclif/core/tree/main/src/help
